pkg/providers/ovirt/validation/validators/storage-mapping-validator.go

```diff
@@ -27,34 +27,66 @@ func NewStorageMappingValidator(provider StorageClassProvider) StorageMappingVal
 	}
 }
 
-// ValidateStorageMapping validates storage domain mapping
-func (v *StorageMappingValidator) ValidateStorageMapping(attachments []*ovirtsdk.DiskAttachment, mapping *[]v2vv1alpha1.ResourceMappingItem) []ValidationFailure {
+// ValidateStorageMapping validates storage domain mapping and disk mapping
+func (v *StorageMappingValidator) ValidateStorageMapping(
+	attachments []*ovirtsdk.DiskAttachment,
+	storageMapping *[]v2vv1alpha1.ResourceMappingItem,
+	diskMapping *[]v2vv1alpha1.ResourceMappingItem,
+) []ValidationFailure {
 	var failures []ValidationFailure
 	// Check whether mapping for storage is required and was provided
-	if mapping == nil && len(attachments) > 0 {
+	if storageMapping == nil && diskMapping == nil && len(attachments) > 0 {
```

Comment:
is it really that bad to have both missing? I guess, it can still be used to send empty mapping if the user doesn't want to specify storage classes (or depend on the default one).

It could be just more convenient to send it without the mapping

Reply:
I think it is better not to send it if is empty.